setup.py

```diff
@@ -58,8 +58,8 @@
       python_requires='>=2.7,!=3.0,!=3.1,!=3.2',
       setup_requires=['katversion'],
       use_katversion=True,
-      install_requires=['numpy', 'katpoint', 'h5py>=2.3',
-                        'katsdptelstate[rdb]', 'dask[array] >= 0.18.2', 'numba',
+      install_requires=['numpy', 'katpoint', 'h5py >= 2.3', 'numba',
+                        'katsdptelstate[rdb] >= 0.7', 'dask[array] >= 0.18.2',
```

Comment:
Does it still work with dask 0.18.2, or are we using any newer features?

Reply:
I can't see any newer features. I am still keen to resolve the `HighLevelGraph` warning soon for the record, as I don't like scaring the users.